webapp/public/styles.css

```
@@ -0,0 +1,192 @@
+
+.result pre {
+  margin: 0;
+  padding: 18px;
+  max-height: 60vh;
+  overflow: auto;
+  background: #0a0f24;
+  font-family: ui-monospace, SFMono-Regular, Menlo, Consolas, 'Liberation Mono', monospace;
+  font-size: 0.875rem;
+  line-height: 1.5;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.6.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)